pkgs/development/rocm-modules/5/rocprofiler/default.nix

```diff
@@ -79,16 +93,31 @@ stdenv.mkDerivation (finalAttrs: {
   postPatch = ''
     patchShebangs .
```

Comment:
not sure how feasible it is to narrow this down

Reply:
There's like two directories with a few files that we can cut out.
IMO not worth narrowing down.

Reply:
Actually scratch that, it's literally only ``build.sh`` that we could cut out.
Everything else should probably stay.